src/components/spinner/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+
+import Loader from 'react-loader';
+import SC from './styled';
+
+export default class Spinner extends Component {
+  render() {
+    return (
+      <SC.Spinner>
+        <div>
+          <Loader>
```

Comment:
Hvorfor ha med Loader? Den er avhengig av state for å fungere. Ellers kan du fjerne hele Loader og dependency.

Reply:
Ok, da tar jeg bort den. :-) Trodde den var nødvendig for å styre dette at spinneren ble borte når siden var lastet, men dette tar vi oss kanskje av utenfor spinneren?